Source/ASDisplayNode+Layout.mm

```diff
@@ -143,19 +144,19 @@ - (NSString *)asciiArtName
 
 @implementation ASDisplayNode (ASLayout)
 
-- (void)setLayoutSpecBlock:(ASLayoutSpecBlock)layoutSpecBlock
+- (ASLayoutType)layoutType
```

Comment:
Good idea! What do you think about ASLayoutEngineType instead?

ASLayoutType sounds a bit too close to describing the actually-produced ASLayout objects.